tests/test_hamiltonian_sparse.py

```diff
@@ -59,7 +59,7 @@ def test_hamiltionan_expectation(self, qubit_device_3_wires, tol):
 
         dev.syncH2D()
         Hmat = qml.utils.sparse_hamiltonian(H)
-        H_sparse = qml.SparseHamiltonian(Hmat, wires=3)
+        H_sparse = qml.SparseHamiltonian(Hmat, wires=range(3))
```

Comment:
Good catch, here!
Could you please go to the test definitions and change:
`def test_hamiltionan_expectation(self, qubit_device_3_wires, tol):`
to
`def test_hamiltonian_expectation(self, qubit_device_3_wires, tol):`?
I can't reach there to comment.

Reply:
Thanks Amintor, and done.